### PR TITLE
Add Cufflinks

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

File: recipes/cufflinks/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - plotting
```
Comment (CJ-Wright) [inline recipes/cufflinks/meta.yaml]: I don't think this package currently exists. Did this build on your local? Could this maybe be plotly (or not needed?)

**Comment by bgruening** [comment]
@sannykr there is already a package called cufflinks downstream: https://bioconda.github.io/recipes/cufflinks/README.html

Can we name this one cufflinks-py or similar?

**Comment by sannykr** [comment]
@bgruening : Sure. I will do that. Thanks

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge if everything looks okay

**Pull request merged**